tests/language-feature/bitfield/zero-width.slang

```diff
@@ -0,0 +1,28 @@
+//TEST(compute):COMPARE_COMPUTE(filecheck-buffer=CHECK):-cpu -output-using-type
+
+// CHECK:      123
+// CHECK-NEXT: 4567
+// CHECK-NEXT: 1
+// CHECK-NEXT: 0
+
+//TEST_INPUT:ubuffer(data=[0 0 0 0], stride=4):out,name=outputBuffer
+RWStructuredBuffer<uint> outputBuffer;
+
+struct S {
+    int foo : 8;
+    int breaker : 0;
```

Comment:
What's the semantics of a 0bit field?

Reply:
It only has one value (0), and indicates that the next bits shouldn't be part of an existing field. I dislike very much how it's this sort of magic inconsistent thing going on, but it's consistent with how my C compiler does things at least...